django_tables2/export/views.py

```diff
@@ -22,23 +22,31 @@ class Table(tables.Table):
                     name = tables.Column()
                     buttons = tables.TemplateColumn(exclude_from_export=True, template_name=...)
         export_formats (iterable): export formats to render a set of buttons in the template.
+        dataset_kwargs (dictionary): passed as `**kwargs` to `tablib.Dataset` constructor::
+
+            dataset_kwargs = {'tite': 'My custom tab title'}
     """
 
     export_class = TableExport
     export_name = "table"
     export_trigger_param = "_export"
     exclude_columns = ()
+    dataset_kwargs = None
```

Comment:
I'd say adding a sensible default for the `title` key might be a nice improvement. Ideas:

 - Something static, i.e. 'export' or 'data'.
 - Something dependent on `Table.Meta.model` if it's defined. (i.e. its verbose name, Persons).

Reply:
Great idea.  default is table.Meta.model._meta.verbose_name_plural, if defined, otherwise "Export Data" .  I added a test for this.